csunplugged/topics/content/en/structure.yaml

```diff
@@ -23,4 +23,4 @@ topics:
     other-resources-md-file:
 
 learning-outcomes: learning-outcomes.yaml
-programming-exercises-difficulty: programming-exercises-difficulty.yaml
+programming-exercises-info: programming-exercises-info.yaml
```

Comment:
Should this be `programming-exercises-structure: programming-exercises-structure.yaml` to be clearer for purpose? And also shows connection to loader.